packages/@azure/servicebus/data-plane/examples/javascript/gettingStarted/sendWithInteractiveAuth.js

```diff
@@ -0,0 +1,56 @@
+/*
+  This sample demonstrates how to create a namespace using TokenCredentials.
```

Comment:
This sample demonstrates how to create a namespace using AAD token credentials gotten from interactive login